package.json

```diff
@@ -87,6 +87,7 @@
     "nsp": "nsp check",
     "posttest": "npm run lint",
     "test": "make bootstrap_zip && mocha test/",
+    "test:server": "mocha --recursive test/server/unit/",
```

Comment:
It could just be renamed to test:server_unit or something. Or does `mocha test/ test/server/unit/` work?